kernel/test/unityIntegration/ecs/math/quaternion.test.ts

```diff
@@ -33,16 +33,13 @@ const results = {
   staticSlerp04: '(0.0, 0.3, 0.0, 0.9)',
   eulerAngles01: '(10.0, 10.0, 10.0)',
   eulerAngles02: '(0.0, 90.0, 0.0)',
-  eulerAngles03: '(80.0, 190.0, 220.0)',
-  eulerAngles04: '(0.0, 10.0, 0.0)',
+  eulerAngles03: '(80.0, -170.0, -140.1)', // In Unity we get (80.0, 190.0, 220.0), but both represent the same rotation -> 360 - 170 = 190; 360 - 140 = 220;
+  eulerAngles04: '(-0.1, 10.0, 0.0)',
   normalized01: '(0.1, 0.1, 0.1, 1.0)',
   normalized02: '(0.0, 0.7, 0.0, 0.7)',
   normalized03: '(-0.7, 0.2, -0.2, -0.6)',
   normalized04: '(0.0, -0.1, 0.0, -1.0)',
-  setFromToRotation01: '(0.0, 0.0, 0.0, 1.0)',
-  setFromToRotation02: '(0.0, 0.7, 0.0, 0.7)',
-  setFromToRotation03: '(0.0, 0.0, 0.4, 0.9)',
-  setFromToRotation04: '(0.4, 0.0, 0.0, 0.9)'
```

Comment:
Why were these deleted?

Reply:
we weren't using them, so there is no calculation showing the original vector nor the rotation that was applied...
I could create 3 new ones, but I already spent more than 1 week with this issue.

Reply:
Epic, I'll add at least 3 more tests on `setFromToRotation` then!